pkg/components/components.go

```diff
@@ -57,6 +57,7 @@ func NewKeplerNamespace() *corev1.Namespace {
 				//   allowPrivilegeEscalation != false (container "kepler-exporter" must set
 				//   securityContext.allowPrivilegeEscalation=false),
 				"pod-security.kubernetes.io/enforce": "privileged",
+				"openshift.io/cluster-monitoring":    "true",
```

Comment:
@BGrasnick, we shouldn't use incluster monitoring for operator that are not part of the platform itself. This reserved for only those operators. Rest of the projects / operators must use UWM or some other solution 

cc: @simonpasquier, please correct me if I am wrong 🙏

Reply:
Correct, patching the namespace with the `openshift.io/cluster-monitoring` label should happen either in the downstream version of the operator or it should be an option of the upstream operator (disabled by default). It is also something that isn't relevant for non-OCP environments.

https://rhobs-handbook.netlify.app/products/openshiftmonitoring/collecting_metrics.md/#collecting-metrics-with-prometheus

Reply:
Thanks @simonpaquier for the link 🙇